lib/pages/settings/view/settings_page.dart

```diff
@@ -115,6 +115,20 @@ class _SettingsPageState extends State<SettingsPage> {
                     S.current.infoReadThePolicy(Utils.packageInfo.appName),
                   ),
                 ),
+                Visibility(
+                  visible: authProvider.currentUserFromCache.isAdmin == true,
+                  child: ListTile(
+                    key: const Key('HandleRequests'),
+                    onTap: () =>
+                        Navigator.of(context).pushNamed(Routes.adminPanel),
+                    title: Text(
+                      S.current.settingsItemAdmin,
+                    ),
+                    subtitle: Text(
+                      S.current.infoAdmin,
+                    ),
```

Comment:
Nit: when the widget is as simple as this and fits on one line, omit the trailing comma so it can be formatted like:
```
title: Text(S.current.settingsItemAdmin),
```